### DISCUSSION TITLE
Allow any emoji reaction

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#16643] [General]
As a user, I want to be able to react with any emoji, not just six default options, similar to Discord.

We should implement Twemoji to be consistant, because the desktop app already uses Twemoji.

### Acceptance criteria:
* Allow users to react with any emoji
* Implement Twemoji on mobile

### Nice to have, not required
* User-generated custom emojis that users can create in-app

**Comment by ibrkhalil** [comment]
Maybe a plus icon as a 6th item in the reactions options that shows the whole twemoji emoji list ?

**Comment by du82** [comment]
> Maybe a plus icon as a 6th item in the reactions options that shows the whole twemoji emoji list ?

I think this is a good idea
